utils/property_map.go

```diff
@@ -0,0 +1,49 @@
+package utils
+
+import (
+	"database/sql/driver"
+	"encoding/json"
+	"errors"
+)
+
+type PropertyMap map[string]interface{}
+
+//Value - simply returns the JSON-encoded representation of the struct.
+func (p PropertyMap) Value() (driver.Value, error) {
+	j, err := json.Marshal(p)
+	return j, err
+}
+
+//Scan - simply decodes a JSON-encoded value into the struct fields.
+func (p *PropertyMap) Scan(src interface{}) error {
```

Comment:
We don't seem to be using this function anywhere 🤔 
We could add functions to the code base as and when we use it 🤔